prow/cmd/deck/main.go

```diff
@@ -163,13 +163,20 @@ var (
 type traceResponseWriter struct {
 	http.ResponseWriter
 	statusCode int
+	size       int
 }
 
 func (trw *traceResponseWriter) WriteHeader(code int) {
 	trw.statusCode = code
 	trw.ResponseWriter.WriteHeader(code)
 }
 
+func (trw *traceResponseWriter) Write(data []byte) (int, error) {
+	size, err := trw.ResponseWriter.Write(data)
+	trw.size = size
```

Comment:
Is it possible to write more than once? It would be an accumulative value if yes.